Dockerfile

```diff
@@ -0,0 +1,63 @@
+# SOURCE AND CREDITS: https://github.com/chrishah/samtools-docker/blob/master/Dockerfile
+
+FROM alpine:latest
+LABEL version="v1.0" description="A docekr image for the Pharmacogenetics ANalysis Pipeline" maintainer="Graeme Ford <Graeme.Ford@tuks.co.za>"
```

Comment:
Change spelling of "docker" and "analysis"

Reply:
The spelling corrections have been made. Thank you very much for catching that!